### DISCUSSION TITLE
C++ export

### DISCUSSION THREAD

**Discussion by pfeatherstone** [tinygrad/tinygrad#1107] [Ideas]
Do you plan on supporting some kind of export for c++ inference ? Maybe an ONNX export ?

**Comment by pfeatherstone** [comment]
Or maybe some crazy Eigen export

**Comment by geohot** [comment]
See imagenet to C compile test

**Reply by pfeatherstone** [reply]
Sorry, i can't find a unit test that showcases an export to C functionality.

**Comment by python273** [comment]
https://github.com/tinygrad/tinygrad/blob/master/examples/compile_efficientnet.py
